_sass/components/_nav.scss

```diff
@@ -162,9 +162,10 @@ nav .usa-search {
 .usa-search--small [type=submit], 
 .usa-search--small .usa-search__submit {
   background-image: none;
-  width: 5rem;
+  width: auto;
   font-family: $sans-serif-font-family;
   font-size: ($base-font-size - 2);
+  text-align: center;
```

Comment:
Is this doing anything if the width is now also `auto`, or could we remove it?

```suggestion
```

Reply:
Good call! I'll remove it in the next PR.